src/ApiPort.VisualStudio/Models/OptionsModel.cs

```diff
@@ -61,24 +61,13 @@ public IList<TargetPlatform> Platforms
         public string OutputDirectory
         {
             get { return _outputDirectory; }
-            set
-            {
-                UpdateProperty(ref _outputDirectory, value);
```

Comment:
Good point - I've updated this so it will check for null or whitespace and default to a reasonable place